src/phoenix/trace/otel/utils.py

```diff
@@ -0,0 +1,372 @@
+import json
```

Comment:
minor directory nit: I think trace already implies otel kinda so I'm not sure we need the nesting. Just a thought.